Error when trying to build for iOS

Hi,

when I try to build my app with this as a dependency the build fails with the following error:

```

Showing Recent Issues
Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the iosApp editor.

Ld /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app/iosApp normal (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -target arm64-apple-ios14.1-simulator -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.2.sdk -L/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator -F/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator -F/Users/moritz/AndroidStudioProjects/TestProject/iosApp/Pods/../../shared/build/cocoapods/framework -F/Users/moritz/AndroidStudioProjects/TestProject/iosApp/Pods/webrtc-kmp/framework -F/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/XCFrameworkIntermediates/webrtc-kmp -filelist /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/arm64/iosApp.LinkFileList -Xlinker -rpath -Xlinker /usr/lib/swift -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -Xlinker -rpath -Xlinker @executable_path/Frameworks -dead_strip -Xlinker -object_path_lto -Xlinker /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/arm64/iosApp_lto.o -Xlinker -export_dynamic -Xlinker -no_deduplicate -Xlinker -objc_abi_version -Xlinker 2 -fobjc-link-runtime -L/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator -L/usr/lib/swift -Xlinker -add_ast_path -Xlinker /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/arm64/iosApp.swiftmodule -ObjC -lc -lc++ -lsqlite3 -lstdc++ -framework AVFoundation -framework AudioToolbox -framework CoreAudio -framework CoreGraphics -framework CoreVideo -framework GLKit -framework VideoToolbox -framework WebRTC -framework shared -Xlinker -sectcreate -Xlinker __TEXT -Xlinker __entitlements -Xlinker /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.app-Simulated.xcent -framework Pods_iosApp -Xlinker -no_adhoc_codesign -Xlinker -dependency_info -Xlinker /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/arm64/iosApp_dependency_info.dat -o /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app/iosApp

ld: warning: ignoring file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/WebRTC.framework/WebRTC, missing required architecture arm64 in file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/WebRTC.framework/WebRTC (1 slices)
ld: in /Users/moritz/AndroidStudioProjects/TestProject/shared/build/cocoapods/framework/shared.framework/shared(dummy_vers.o), building for iOS Simulator, but linking in object file built for iOS, file '/Users/moritz/AndroidStudioProjects/TestProject/shared/build/cocoapods/framework/shared.framework/shared' for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I am quite sure I have set everything up correctly.
This is my podfile:
```ruby
target 'iosApp' do
  use_frameworks!
  platform :ios, '14.1'
  pod 'shared', :path => '../shared'
  pod 'webrtc-kmp', :git => 'git@github.com:shepeliev/webrtc-kmp.git', :tag => 'v0.89.6'
end
```

This is my shared build.gradle:
```gradle
plugins {
    kotlin("multiplatform")
    kotlin("native.cocoapods")
    id("com.android.library")
}

version = "1.0"

kotlin {
    android()
    iosX64()
    iosArm64()
    iosSimulatorArm64()
    jvm()

    cocoapods {
        summary = "Some description for the Shared Module"
        homepage = "Link to the Shared Module homepage"
        ios.deploymentTarget = "14.1"
        podfile = project.file("../iosApp/Podfile")
        framework {
            baseName = "shared"
        }
    }

    ios {
        binaries
            .filterIsInstance<org.jetbrains.kotlin.gradle.plugin.mpp.Framework>()
            .forEach {
                it.transitiveExport = true
                it.export("com.shepeliev:webrtc-kmp:0.89.6")
            }
    }
    
    sourceSets {
        val commonMain by getting
        val commonTest by getting {
            dependencies {
                implementation(kotlin("test"))
            }
        }

        val clientMain by creating
        val clientTest by creating

        val androidMain by getting {
            dependsOn(clientMain)

            dependencies {
                api("com.shepeliev:webrtc-kmp:0.89.6")
                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.0-native-mt")
            }
        }
        val androidTest by getting {
            dependsOn(clientTest)
        }
        val iosMain by getting
        val iosTest by getting
        val jvmMain by getting
        val jvmTest by getting
    }
}

android {
    compileSdk = 32
    sourceSets["main"].manifest.srcFile("src/androidMain/AndroidManifest.xml")
    defaultConfig {
        minSdk = 21
        targetSdk = 32
    }
}
```

I have made sure that I am opening the *.xcworkspace* file instead of the *.xcodeproj* one. I have also tried these two commands in the `iosApp` directory:
```
pod deintegrate
pod install
```
They ran successfully.

From the error message I am assuming this could be because I am using a mac with an M1 chip. What do you think?

Thanks for the help. I tried what the guy was suggesting but it is still not working...

When I try to build the iosApp using Android Studio I get this error:

```
FAILURE: Build failed with an exception.

* What went wrong:
Task 'syncFramework' not found in project ':shared'.

* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
Command PhaseScriptExecution failed with a nonzero exit code

warning: Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the iosApp editor. (in target 'iosApp' from project 'iosApp')
warning: None of the architectures in ARCHS (arm64) are valid. Consider setting ARCHS to $(ARCHS_STANDARD) or updating it to include at least one value from VALID_ARCHS (arm64, arm64e, i386, x86_64) which is not in EXCLUDED_ARCHS (arm64). (in target 'iosApp' from project 'iosApp')
** BUILD FAILED **


The following build commands failed:
	PhaseScriptExecution [CP-User]\ Build\ shared /Users/moritz/AndroidStudioProjects/CollaborativeStudying/build/ios/Pods.build/Debug-iphonesimulator/shared.build/Script-4552119A071AC6BAB7327E6434237EC3.sh (in target 'shared' from project 'Pods')
(1 failure)
```

However, when I try building using XCode I get a different error:
```

Showing Recent Messages
Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the iosApp editor.

MkDir /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    /bin/mkdir -p /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app

MkDir /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app/Frameworks (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    /bin/mkdir -p /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app/Frameworks

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources/Entitlements-Simulated.plist (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources/Entitlements-Simulated.plist

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources/Entitlements.plist (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources/Entitlements.plist

ProcessProductPackaging "" /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.app-Simulated.xcent (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    
    Entitlements:
    
    {
    "application-identifier" = "FAKETEAMID.orgIdentifier.iosApp";
    "keychain-access-groups" =     (
        "FAKETEAMID.orgIdentifier.iosApp"
    );
}
    
    builtin-productPackagingUtility -entitlements -format xml -o /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.app-Simulated.xcent

ProcessProductPackaging "" /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.app.xcent (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    
    Entitlements:
    
    {
    "com.apple.security.get-task-allow" = 1;
}
    
    builtin-productPackagingUtility -entitlements -format xml -o /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.app.xcent

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.hmap (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.hmap

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-project-headers.hmap (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-project-headers.hmap

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-generated-files.hmap (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-generated-files.hmap

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-own-target-headers.hmap (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-own-target-headers.hmap

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-all-non-framework-target-headers.hmap (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-all-non-framework-target-headers.hmap

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-all-target-headers.hmap (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-all-target-headers.hmap

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/all-product-headers.yaml (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/all-product-headers.yaml

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Script-78A4F8147C973C9B8D58710E.sh (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Script-78A4F8147C973C9B8D58710E.sh

PhaseScriptExecution [CP]\ Check\ Pods\ Manifest.lock /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Script-78A4F8147C973C9B8D58710E.sh (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    /bin/sh -c /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Script-78A4F8147C973C9B8D58710E.sh

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.LinkFileList (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.LinkFileList

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp-OutputFileMap.json (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp-OutputFileMap.json

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.SwiftFileList (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.SwiftFileList

CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    export DEVELOPER_DIR\=/Applications/Xcode.app/Contents/Developer
    export SDKROOT\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.2.sdk
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc -incremental -module-name iosApp -Onone -enable-batch-mode -enforce-exclusivity\=checked @/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.SwiftFileList -DDEBUG -sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.2.sdk -target x86_64-apple-ios14.1-simulator -g -module-cache-path /Users/moritz/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -Xfrontend -serialize-debugging-options -enable-testing -index-store-path /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Index/DataStore -swift-version 5 -I /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator -F /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator -F /Users/moritz/AndroidStudioProjects/CollaborativeStudying/shared/build/cocoapods/framework -F /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp/Pods/webrtc-kmp/framework -F /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/XCFrameworkIntermediates/webrtc-kmp -c -j10 -output-file-map /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp-OutputFileMap.json -parseable-output -serialize-diagnostics -emit-dependencies -emit-module -emit-module-path /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftmodule -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/swift-overrides.hmap -Xcc -iquote -Xcc /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-generated-files.hmap -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-own-target-headers.hmap -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-all-target-headers.hmap -Xcc -iquote -Xcc /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-project-headers.hmap -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/include -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources-normal/x86_64 -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources/x86_64 -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources -Xcc -DDEBUG\=1 -Xcc -DCOCOAPODS\=1 -emit-objc-header -emit-objc-header-path /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp-Swift.h -working-directory /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp

Ld /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app/iosApp normal (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -target x86_64-apple-ios14.1-simulator -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.2.sdk -L/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator -F/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator -F/Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp/Pods/../../shared/build/cocoapods/framework -F/Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp/Pods/webrtc-kmp/framework -F/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/XCFrameworkIntermediates/webrtc-kmp -filelist /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.LinkFileList -Xlinker -rpath -Xlinker /usr/lib/swift -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -Xlinker -rpath -Xlinker @executable_path/Frameworks -dead_strip -Xlinker -object_path_lto -Xlinker /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp_lto.o -Xlinker -export_dynamic -Xlinker -no_deduplicate -Xlinker -objc_abi_version -Xlinker 2 -fobjc-link-runtime -L/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator -L/usr/lib/swift -Xlinker -add_ast_path -Xlinker /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftmodule -ObjC -lc -lc++ -lsqlite3 -lstdc++ -framework AVFoundation -framework AudioToolbox -framework CoreAudio -framework CoreGraphics -framework CoreVideo -framework GLKit -framework VideoToolbox -framework WebRTC -framework shared -Xlinker -sectcreate -Xlinker __TEXT -Xlinker __entitlements -Xlinker /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.app-Simulated.xcent -framework Pods_iosApp -Xlinker -no_adhoc_codesign -Xlinker -dependency_info -Xlinker /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp_dependency_info.dat -o /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app/iosApp

ld: warning: directory not found for option '-F/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/XCFrameworkIntermediates/webrtc-kmp'
ld: framework not found WebRTC
clang: error: linker command failed with exit code 1 (use -v to see invocation)

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftmodule /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64.swiftmodule (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks -rename /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftmodule /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64.swiftmodule

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftmodule /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64-apple-ios-simulator.swiftmodule (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks -rename /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftmodule /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64-apple-ios-simulator.swiftmodule

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp-Swift.h /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources/iosApp-Swift.h (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp-Swift.h /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftdoc /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64-apple-ios-simulator.swiftdoc (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks -rename /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftdoc /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64-apple-ios-simulator.swiftdoc

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftdoc /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64.swiftdoc (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks -rename /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftdoc /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64.swiftdoc

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftsourceinfo /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/Project/x86_64.swiftsourceinfo (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks -rename /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftsourceinfo /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/Project/x86_64.swiftsourceinfo

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftsourceinfo /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/Project/x86_64-apple-ios-simulator.swiftsourceinfo (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/CollaborativeStudying/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks -rename /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftsourceinfo /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/Project/x86_64-apple-ios-simulator.swiftsourceinfo


```

These are all warnings except "Framework not found WebRTC".

I found this article about GoogleWebRTC not being compatible with M1, do you think that could help? https://tokhirov-mukhammadjon.medium.com/build-webrtc-manually-a220ebec59bf

Even if this would help, I would honestly not quite know how to approach this since I am still quite new to iOS.

Thanks for the help. I tried what the guy was suggesting but it is still not working...

When I try to build the iosApp using Android Studio I get this error:

```
FAILURE: Build failed with an exception.

* What went wrong:
Task 'syncFramework' not found in project ':shared'.

* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
Command PhaseScriptExecution failed with a nonzero exit code

warning: Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the iosApp editor. (in target 'iosApp' from project 'iosApp')
warning: None of the architectures in ARCHS (arm64) are valid. Consider setting ARCHS to $(ARCHS_STANDARD) or updating it to include at least one value from VALID_ARCHS (arm64, arm64e, i386, x86_64) which is not in EXCLUDED_ARCHS (arm64). (in target 'iosApp' from project 'iosApp')
** BUILD FAILED **


The following build commands failed:
	PhaseScriptExecution [CP-User]\ Build\ shared /Users/moritz/AndroidStudioProjects/TestProject/build/ios/Pods.build/Debug-iphonesimulator/shared.build/Script-4552119A071AC6BAB7327E6434237EC3.sh (in target 'shared' from project 'Pods')
(1 failure)
```

However, when I try building using XCode I get a different error:
```

Showing Recent Messages
Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the iosApp editor.

MkDir /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    /bin/mkdir -p /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app

MkDir /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app/Frameworks (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    /bin/mkdir -p /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app/Frameworks

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources/Entitlements-Simulated.plist (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources/Entitlements-Simulated.plist

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources/Entitlements.plist (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources/Entitlements.plist

ProcessProductPackaging "" /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.app-Simulated.xcent (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    
    Entitlements:
    
    {
    "application-identifier" = "FAKETEAMID.orgIdentifier.iosApp";
    "keychain-access-groups" =     (
        "FAKETEAMID.orgIdentifier.iosApp"
    );
}
    
    builtin-productPackagingUtility -entitlements -format xml -o /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.app-Simulated.xcent

ProcessProductPackaging "" /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.app.xcent (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    
    Entitlements:
    
    {
    "com.apple.security.get-task-allow" = 1;
}
    
    builtin-productPackagingUtility -entitlements -format xml -o /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.app.xcent

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.hmap (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.hmap

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-project-headers.hmap (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-project-headers.hmap

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-generated-files.hmap (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-generated-files.hmap

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-own-target-headers.hmap (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-own-target-headers.hmap

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-all-non-framework-target-headers.hmap (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-all-non-framework-target-headers.hmap

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-all-target-headers.hmap (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-all-target-headers.hmap

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/all-product-headers.yaml (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/all-product-headers.yaml

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Script-78A4F8147C973C9B8D58710E.sh (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Script-78A4F8147C973C9B8D58710E.sh

PhaseScriptExecution [CP]\ Check\ Pods\ Manifest.lock /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Script-78A4F8147C973C9B8D58710E.sh (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    /bin/sh -c /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Script-78A4F8147C973C9B8D58710E.sh

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.LinkFileList (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.LinkFileList

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp-OutputFileMap.json (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp-OutputFileMap.json

WriteAuxiliaryFile /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.SwiftFileList (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    write-file /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.SwiftFileList

CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    export DEVELOPER_DIR\=/Applications/Xcode.app/Contents/Developer
    export SDKROOT\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.2.sdk
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc -incremental -module-name iosApp -Onone -enable-batch-mode -enforce-exclusivity\=checked @/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.SwiftFileList -DDEBUG -sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.2.sdk -target x86_64-apple-ios14.1-simulator -g -module-cache-path /Users/moritz/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -Xfrontend -serialize-debugging-options -enable-testing -index-store-path /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Index/DataStore -swift-version 5 -I /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator -F /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator -F /Users/moritz/AndroidStudioProjects/TestProject/shared/build/cocoapods/framework -F /Users/moritz/AndroidStudioProjects/TestProject/iosApp/Pods/webrtc-kmp/framework -F /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/XCFrameworkIntermediates/webrtc-kmp -c -j10 -output-file-map /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp-OutputFileMap.json -parseable-output -serialize-diagnostics -emit-dependencies -emit-module -emit-module-path /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftmodule -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/swift-overrides.hmap -Xcc -iquote -Xcc /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-generated-files.hmap -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-own-target-headers.hmap -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-all-target-headers.hmap -Xcc -iquote -Xcc /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp-project-headers.hmap -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/include -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources-normal/x86_64 -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources/x86_64 -Xcc -I/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources -Xcc -DDEBUG\=1 -Xcc -DCOCOAPODS\=1 -emit-objc-header -emit-objc-header-path /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp-Swift.h -working-directory /Users/moritz/AndroidStudioProjects/TestProject/iosApp

Ld /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app/iosApp normal (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -target x86_64-apple-ios14.1-simulator -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.2.sdk -L/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator -F/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator -F/Users/moritz/AndroidStudioProjects/TestProject/iosApp/Pods/../../shared/build/cocoapods/framework -F/Users/moritz/AndroidStudioProjects/TestProject/iosApp/Pods/webrtc-kmp/framework -F/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/XCFrameworkIntermediates/webrtc-kmp -filelist /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.LinkFileList -Xlinker -rpath -Xlinker /usr/lib/swift -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -Xlinker -rpath -Xlinker @executable_path/Frameworks -dead_strip -Xlinker -object_path_lto -Xlinker /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp_lto.o -Xlinker -export_dynamic -Xlinker -no_deduplicate -Xlinker -objc_abi_version -Xlinker 2 -fobjc-link-runtime -L/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator -L/usr/lib/swift -Xlinker -add_ast_path -Xlinker /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftmodule -ObjC -lc -lc++ -lsqlite3 -lstdc++ -framework AVFoundation -framework AudioToolbox -framework CoreAudio -framework CoreGraphics -framework CoreVideo -framework GLKit -framework VideoToolbox -framework WebRTC -framework shared -Xlinker -sectcreate -Xlinker __TEXT -Xlinker __entitlements -Xlinker /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/iosApp.app-Simulated.xcent -framework Pods_iosApp -Xlinker -no_adhoc_codesign -Xlinker -dependency_info -Xlinker /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp_dependency_info.dat -o /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.app/iosApp

ld: warning: directory not found for option '-F/Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/XCFrameworkIntermediates/webrtc-kmp'
ld: framework not found WebRTC
clang: error: linker command failed with exit code 1 (use -v to see invocation)

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftmodule /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64.swiftmodule (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks -rename /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftmodule /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64.swiftmodule

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftmodule /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64-apple-ios-simulator.swiftmodule (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks -rename /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftmodule /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64-apple-ios-simulator.swiftmodule

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp-Swift.h /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources/iosApp-Swift.h (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp-Swift.h /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/DerivedSources

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftdoc /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64-apple-ios-simulator.swiftdoc (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks -rename /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftdoc /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64-apple-ios-simulator.swiftdoc

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftdoc /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64.swiftdoc (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks -rename /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftdoc /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/x86_64.swiftdoc

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftsourceinfo /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/Project/x86_64.swiftsourceinfo (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks -rename /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftsourceinfo /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/Project/x86_64.swiftsourceinfo

PBXCp /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftsourceinfo /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/Project/x86_64-apple-ios-simulator.swiftsourceinfo (in target 'iosApp' from project 'iosApp')
    cd /Users/moritz/AndroidStudioProjects/TestProject/iosApp
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks -rename /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Intermediates.noindex/iosApp.build/Debug-iphonesimulator/iosApp.build/Objects-normal/x86_64/iosApp.swiftsourceinfo /Users/moritz/Library/Developer/Xcode/DerivedData/iosApp-gfbujjgcrbxffubzfovqsagohwuo/Build/Products/Debug-iphonesimulator/iosApp.swiftmodule/Project/x86_64-apple-ios-simulator.swiftsourceinfo


```

These are all warnings except "Framework not found WebRTC".

I found this article about GoogleWebRTC not being compatible with M1, do you think that could help? https://tokhirov-mukhammadjon.medium.com/build-webrtc-manually-a220ebec59bf

Even if this would help, I would honestly not quite know how to approach this since I am still quite new to iOS.

--- a/sample/README.md
+++ b/sample/README.md
@@ -7,7 +7,7 @@ Demo application for WebRTC KMP.
 ```bash
 git clone https://github.com/shepeliev/webrtc-kmp.git
 cd webrtc-kmp
-carthage update --use--xcframeworks
+carthage update --use-xcframeworks
 ```
 
 ### Android
@@ -15,7 +15,7 @@ carthage update --use--xcframeworks
 Run Android emulator or connect real device.
 
 ```bash
-./graldew installDebug
+./gradlew installDebug
 ```
 
 ### iOS

--- a/sample/shared/build.gradle.kts
+++ b/sample/shared/build.gradle.kts
@@ -1,4 +1,5 @@
 import org.jetbrains.kotlin.gradle.plugin.mpp.apple.XCFramework
+import org.jetbrains.kotlin.gradle.plugin.mpp.apple.XCFrameworkConfig
 
 plugins {
     id("multiplatform-setup")
@@ -7,38 +8,13 @@ plugins {
 kotlin {
     val xcf = XCFramework()
 
-    ios {
-        val frameworks = getFrameworks(konanTarget).filterKeys { it != "WebRTC" }
+    ios { configureIos(xcf) }
+    iosSimulatorArm64 { configureIos(xcf) }
 
-        compilations.getByName("main") {
-            cinterops.create("FirebaseCore") {
-                frameworks.forEach { (framework, path) ->
-                    compilerOpts("-framework", framework, "-F$path")
-                }
-            }
-
-            cinterops.create("FirebaseFirestore") {
-                frameworks.forEach { (framework, path) ->
-                    compilerOpts("-framework", framework, "-F$path")
-                }
-            }
-        }
-
-        binaries.framework {
-            baseName = "shared"
-            xcf.add(this)
-
-            export(project(":webrtc-kmp"))
-            export(deps.decompose)
-            transitiveExport = true
-            isStatic = true
-
-            frameworks.forEach { (framework, path) ->
-                linkerOpts("-framework", framework, "-F$path")
-            }
-
-            linkerOpts("-ObjC")
-        }
+    sourceSets {
+        val iosMain by getting
+        val iosSimulatorArm64Main by getting
+        iosSimulatorArm64Main.dependsOn(iosMain)
     }
 }
 
@@ -55,4 +31,38 @@ dependencies {
     androidMainImplementation(deps.firebase.firestore)
     androidMainImplementation(deps.kotlin.coroutinesPlayServices)
     jsMainImplementation(npm("firebase", version = "9.9.1"))
+}
+
+fun org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget.configureIos(xcf: XCFrameworkConfig) {
+    val frameworks = getFrameworks(konanTarget).filterKeys { it != "WebRTC" }
+
+    compilations.getByName("main") {
+        cinterops.create("FirebaseCore") {
+            frameworks.forEach { (framework, path) ->
+                compilerOpts("-framework", framework, "-F$path")
+            }
+        }
+
+        cinterops.create("FirebaseFirestore") {
+            frameworks.forEach { (framework, path) ->
+                compilerOpts("-framework", framework, "-F$path")
+            }
+        }
+    }
+
+    binaries.framework {
+        baseName = "shared"
+        xcf.add(this)
+
+        export(project(":webrtc-kmp"))
+        export(deps.decompose)
+        transitiveExport = true
+        isStatic = true
+
+        frameworks.forEach { (framework, path) ->
+            linkerOpts("-framework", framework, "-F$path")
+        }
+
+        linkerOpts("-ObjC")
+    }
 }

--- a/webrtc-kmp/build.gradle.kts
+++ b/webrtc-kmp/build.gradle.kts
@@ -15,6 +15,16 @@ kotlin {
     }
     iosSimulatorArm64 { configureIos() }
     ios { configureIos() }
+
+    sourceSets {
+        val iosMain by getting
+        val iosSimulatorArm64Main by getting
+        iosSimulatorArm64Main.dependsOn(iosMain)
+
+        val iosTest by getting
+        val iosSimulatorArm64Test by getting
+        iosSimulatorArm64Test.dependsOn(iosTest)
+    }
 }
 
 android {

--- a/webrtc-kmp/build.gradle.kts
+++ b/webrtc-kmp/build.gradle.kts
@@ -13,26 +13,8 @@ kotlin {
     android {
         publishAllLibraryVariants()
     }
-
-    ios {
-        val frameworks = getFrameworks(konanTarget).filterKeys { it == "WebRTC" }
-
-        compilations.getByName("main") {
-            cinterops.create("WebRTC") {
-                frameworks.forEach { (framework, path) ->
-                    compilerOpts("-framework", framework, "-F$path")
-                }
-            }
-        }
-
-        binaries {
-            getTest("DEBUG").apply {
-                frameworks.forEach { (framework, path) ->
-                    linkerOpts("-framework", framework, "-F$path", "-rpath", "$path", "-ObjC")
-                }
-            }
-        }
-    }
+    iosSimulatorArm64 { configureIos() }
+    ios { configureIos() }
 }
 
 android {
@@ -98,4 +80,24 @@ signing {
 
     useInMemoryPgpKeys(signingKey, signingPassword)
     sign(publishing.publications)
+}
+
+fun org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget.configureIos() {
+    val frameworks = getFrameworks(konanTarget).filterKeys { it == "WebRTC" }
+
+    compilations.getByName("main") {
+        cinterops.create("WebRTC") {
+            frameworks.forEach { (framework, path) ->
+                compilerOpts("-framework", framework, "-F$path")
+            }
+        }
+    }
+
+    binaries {
+        getTest("DEBUG").apply {
+            frameworks.forEach { (framework, path) ->
+                linkerOpts("-framework", framework, "-F$path", "-rpath", "$path", "-ObjC")
+            }
+        }
+    }
 }

